user-org-service/app/controllers/BaseController.java

```diff
@@ -35,25 +50,139 @@ public String getDummyResponse() {
   public CompletionStage<Result> handelRequest() {
     Http.RequestBody requestBody = request().body();
     startTrace("handelRequest");
-    CompletableFuture<String> cf = new CompletableFuture<>();
-    cf.complete(getDummyResponse());
+    CompletableFuture<String> future = new CompletableFuture<>();
+    future.complete(getDummyResponse());
     endTrace("handelRequest");
-    return cf.thenApplyAsync(Results::ok, httpObject.current());
+    return future.thenApplyAsync(Results::ok, httpExecutionContext.current());
   }
 
+  /**
+   * This method will return the current timestamp.
+   *
+   * @return long
+   */
   public long getTimeStamp() {
     return System.currentTimeMillis();
   }
 
+  /**
+   * This method we used to print the logs of starting time of methods
+   *
+   * @param tag
+   */
   public void startTrace(String tag) {
     ProjectLogger.log(
         String.format("%s:%s:started at %s", this.getClass().getSimpleName(), tag, getTimeStamp()),
         LoggerEnum.DEBUG.name());
   }
 
+  /**
+   * This method we used to print the logs of ending time of methods
+   *
+   * @param tag
+   */
   public void endTrace(String tag) {
     ProjectLogger.log(
         String.format("%s:%s:ended at %s", this.getClass().getSimpleName(), tag, getTimeStamp()),
         LoggerEnum.DEBUG.name());
   }
+
+  /**
+   * This method will redirect Response object on the basis of error is present or not present in
+   * response
+   *
+   * @param response
+   * @return CompletionStage<Result>
+   */
+  public CompletionStage<Result> handelResponse(Response response) {
+    return (Boolean) response.get(UserOrgJsonKey.ERROR)
+        ? handelFailureResponse(response)
+        : handelSuccessResponse(response);
+  }
+
+  /**
+   * This method will handel all the success response of Api calls.
+   *
+   * @param response
+   * @return
+   */
+  public CompletionStage<Result> handelSuccessResponse(Response response) {
+    CompletableFuture<String> future = new CompletableFuture<>();
+    String jsonResponse = jsonifyResponseObject(response);
+    if (StringUtils.isNotBlank(jsonResponse)) {
```

Comment:
Lets try check this before. I mean if we could pass jsonified respose to these functions. Also, the apache3 jar is not so useful, if we use it for isNotBlank function.